docs/developer-resources.md

```diff
@@ -0,0 +1,346 @@
+# Developer Resources
+
+This document contains as many helpful instructions for future software engineering senior project teams as I could think of. All of the instructions are written with two assumptions:
+
+1. Unix (Linux/MacOS) operating system. I am 100% aware that there will be future users on Windows, but it wasn't feasible to write everything twice so you will need to adjust accordingly. The main reason this is written with Unix in mind is because the VM runs on Ubuntu. If you use Windows 10 you can look into [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10) although keep in mind that this is not a simple process at all and has a few drawbacks as there's essentially two operating systems that may end up fighting each other at times. I do use it on my desktop though and would definitely recommend it over command prompt.
```

Comment:
I would say `...because the VM the server runs on is Ubuntu` rather than what you have now